### PR TITLE
chore: bump version to 0.8.0

### PR DESCRIPTION
**Pull request by ShahakShama**
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-api/178)

### CODE REVIEW

**Review by ShahakShama — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-api/178)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @ShahakShama and @yair-starkware)

*[a discussion](https://reviewable.io/reviews/starkware-libs/starknet-api/178#-Nq1y68l4PKposWdcabg:-Nq1y68l4PKposWdcabh:bh59818) (no related file):*
wait until #177  is merged

___

**Review by yair-starkware — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/starknet-api/178#-:-Nq1y93S5v60u1936I6B:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @ShahakShama)

**Pull request merged**
